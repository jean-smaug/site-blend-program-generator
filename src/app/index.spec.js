import React from 'react';
import configureStore from 'redux-mock-store';
import renderer from 'react-test-renderer';
import { App } from './index';

describe('app.component', () => {
  let store;
  let tree;
  const mockStore = configureStore();
  const initialState = {
    smoothie: { dayOne: {}, dayTwo: {} },
  };

  beforeEach(() => {
    store = mockStore(initialState);
    tree = renderer.create(<App store={store} />).toJSON();
  });

  it('=== SNAPSHOT ===', () => {
    expect(tree).toMatchSnapshot();
  });
});
