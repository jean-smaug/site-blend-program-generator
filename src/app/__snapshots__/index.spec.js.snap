// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app.component should render App Component 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <div
    className="App"
>
    <div
        className="columns"
    >
        <div
            className="column"
        >
            <Connect(Component) />
        </div>
    </div>
    <div
        className="columns"
    >
        <div
            className="column"
        />
    </div>
</div>,
  "nodes": Array [
    <div
      className="App"
>
      <div
            className="columns"
      >
            <div
                  className="column"
            >
                  <Connect(Component) />
            </div>
      </div>
      <div
            className="columns"
      >
            <div
                  className="column"
            />
      </div>
</div>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 2,
      "_context": Object {},
      "_currentElement": <Unknown
        smoothie={
                Object {
                        "conferences": Array [],
                        "dayOne": Object {},
                        "dayTwo": Object {},
                      }
        }
/>,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StatelessComponent {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {
          "smoothie": Object {
            "conferences": Array [],
            "dayOne": Object {},
            "dayTwo": Object {},
          },
        },
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <div
          className="App"
>
          <div
                    className="columns"
          >
                    <div
                              className="column"
                    >
                              <Connect(Component) />
                    </div>
          </div>
          <div
                    className="columns"
          >
                    <div
                              className="column"
                    />
          </div>
</div>,
        "_debugID": 2,
        "_renderedOutput": <div
          className="App"
>
          <div
                    className="columns"
          >
                    <div
                              className="column"
                    >
                              <Connect(Component) />
                    </div>
          </div>
          <div
                    className="columns"
          >
                    <div
                              className="column"
                    />
          </div>
</div>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <Unknown
    smoothie={
        Object {
            "conferences": Array [],
            "dayOne": Object {},
            "dayTwo": Object {},
          }
    }
/>,
}
`;
